components/tidb_query/src/executor/runner.rs

```diff
@@ -109,15 +110,18 @@ fn build_first_executor<S: Storage + 'static, C: ExecSummaryCollector + 'static>
     mut first: tipb::Executor,
     storage: S,
     ranges: Vec<KeyRange>,
+    context: Arc<EvalConfig>,
```

Comment:
Why use different names here?

Reply:
The word `context` match the field `context` of `ScanExecutor`